test/integration/guide_create_edit_test.rb

```diff
@@ -27,6 +27,34 @@ class GuideCreateEditTest < JavascriptIntegrationTest
     assert_equal "Foo bar", g.title
   end
 
+  should "editing a GuideEdition, and hiding chapter navigation" do
+    guide = FactoryBot.create(:guide_edition,
+                               panopticon_id: @artefact.id,
+                               state: 'draft',
+                               title: "Foo bar #0")
+
+    visit_edition guide
+    refute find('#edition_hide_chapter_navigation').checked?
+    page.execute_script("$('#edition_hide_chapter_navigation').attr('checked', 'checked')")
```

Comment:
@leenagupte got it working 👍

Reply:
It's a bit ugly (unreadable). Can you add a card to our backlog to revisit this test and get it working "properly". That could mean switching to using the headless Chrome browser.

Reply:
Added - https://trello.com/c/7gXatxse/618-improve-hidechapternavigation-test, also added a comment to the file to clarify.

Reply:
Thanks @andysellick!